src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala

```diff
@@ -27,6 +26,20 @@ class FileService(
                    uuidSource: UUIDSource
                  )(implicit val executionContext: ExecutionContext) {
 
+  implicit class FileRowHelper(fr: Option[FileRow]) {
+    def fileType: Option[String] = {
+      if (fr.isDefined) { fr.get.filetype } else None
```

Comment:
This is personal preference so feel free to ignore it but could you simplify this and the others slightly with `fr.flatMap(_.filetype)`